examples/TensorFlow/Train/DeePMD/data/data_0/type.raw

```diff
@@ -0,0 +1,192 @@
+0
```

Comment:
To avoid uploading data to the repo, maybe we can upload it to OSS and download it in Python code.

Reply:
Done. Thanks.